features/old/services/multiservice.feature

```diff
@@ -7,7 +7,7 @@ Feature: Multiservice feature
   Background:
     Given a provider is logged in
     And a default service of provider "foo.3scale.localhost" has name "Fancy API"
-    And a service "Second service" of provider "foo.3scale.localhost"
+    And current domain is the admin domain of provider "foo.3scale.localhost"
```

Comment:
This step is called already in `a provider is logged in`.
```suggestion
```

Reply:
correct